### PR TITLE
Add ability to create unstable images

### PR DESCRIPTION
**Pull request by rybaktomasz**
I've added "unstable" option to manifest schema.
I've also moved release-specific configurations to JSON files as there will be more them - one for each release.
I've also disabled DISTRIBUTION-updates for unstable - it is only available for stable releases.

### CODE REVIEW

**Comment by andsens** [comment]
This is a terrific PR. Nice job! Any chance you could rebase so I can merge without conflicts?
I like the idea of putting release specific info outside the source code, that makes it easier to spot. Maybe there should be a folder for it once we get a few more of those files (or maybe just put it all into one file - per provider).

**Comment by rybaktomasz** [comment]
I've rebased my branch so now it should apply cleanly.

File: common/tasks/network.py [hunk 1]
```diff
@@ -30,11 +30,9 @@ class ConfigureNetworkIF(Task):
...
-		                         'iface eth0 inet dhcp\n'),
-		             'wheezy':  'auto eth0\n'
-		                        'iface eth0 inet dhcp\n'}
+		if_config = []
+		with open('common/tasks/network-configuration.json') as stream:
+			import json
+			if_config = json.loads(stream.read())
 		with open(interfaces_path, 'a') as interfaces:
-			interfaces.write(if_config.get(info.manifest.system['release']))
+			interfaces.write(''.join(if_config.get(info.manifest.system['release'])))
```
Comment (andsens) [inline common/tasks/network.py:38]: You could use `'\n'.join()` and avoid having to type the newlines in the JSON file.

**Comment by andsens** [comment]
Great stuff :-)

**Pull request merged**